apps/antalmanac/src/actions/AppStoreActions.ts

```diff
@@ -12,6 +12,11 @@ import AppStore from '$stores/AppStore';
 import trpc from '$lib/api/trpc';
 import { courseNumAsDecimal } from '$lib/analytics';
 
+export interface CallbackOptions {
```

Comment:
It depends on whether or not we want to reuse these callback options for other functions, i.e. all the helper functions that trigger the snackbar, but I think it's fine to specify that these options are specific to the `copySchedule` function.

```suggestion
export interface CopyScheduleOptions {
```